mbbsd/mail.c

```diff
@@ -1213,6 +1213,32 @@ get_account_sysop(struct Vector *namelist)
         return -1;
 }
 
+void
+show_mail_account_sysop_desc(void)
+{
+    int index = 1;
+    FILE *fp;
+
+    vs_hdr("�b������H�c����");
+
+    if ( (fp = fopen(FN_MAIL_ACCOUNT_SYSOP_DESC, "r")) != NULL) {
+        char line[STRLEN+1];
+        while(fgets(line, sizeof(line), fp)) {
```

Comment:
index is unused in the else block. could you move it here?
for (int i = 0; i < 24 && fgets(...) != NULL; i++) {